source/unified-test-format/unified-test-format.rst

```diff
@@ -195,6 +195,21 @@ error if an attempt is made to store an entity with a name that already exists
 in the map and MUST raise an error if an entity is not found for a name or is
 found but has an unexpected type.
 
+Test runners MUST provide a mechanism to retrieve entities from the entity
+map. There MUST be a way to:
```

Comment:
Per our discussion earlier today, I revised the language pertaining to entity retrieval to specify when the retrieval must be possible and to provide options as to how it might be implemented.